public/schedule.js

```diff
@@ -159,6 +168,48 @@ class schedule_frontend {
 		});
 	}
 
+	/**
+	 * Watch for previews
+	 * @param {String} button
+	 */
+	preview_watch(button) {
```

Comment:
I think that browser side should keep the code to pre ES6, since we have no webpack and should try to keep support for more browsers. Unless you want to implement webpack ;P

Reply:
I tried to make sure anything I used had wide browser support. Other than IE11, which stats say is around 1-3% usage, is using ES6 still a big issue in Sept 2020?

Reply:
I can help you refactor this if you'd like.

Reply:
I've got most of the code rewritten for old browsers.

However, now I'm at the more difficult spot of testing against legacy browsers. Since browsers in the past 5 years should have support for the ES6, which browsers are we looking to support so I can test against them? One problem is, I'm using quite a lot of ES6/new functionality in my code (forEach, spread, default parameters, etc, etc), so some of those are newerish (default params) and will be harder to check for.

Reply:
@haakonnessjoen see above. Which browsers should I test against?

Reply:
Would you be opposed to just hiding the schedule functionality If the browser is updated? That would be a super simple fix. 

If not, I had started working on the other version, like I said earlier